### ISSUE TITLE
XAxis is not correctly sizing height when font size is changed. 

### ISSUE THREAD

**Issue by edwwsw** [ScottPlot/ScottPlot#3436]
![](https://github.com/ScottPlot/ScottPlot/assets/14278121/0306b6c9-ab9e-451f-90b3-42572e209861)

The XAxis has text clipped if you increase the default font size.  Note that the Y-Axis seems to correctly size itself. 

```cs
WpfPlot.Plot.Axes.Left.Label.FontSize = 32;
WpfPlot.Plot.Axes.Bottom.Label.FontSize = 32;
```

Edit:  Occurring in version 5.0.21

**Comment by swharden** [comment]
Thanks for reporting this @edwwsw! Here's a more complete code sample that helps reproduce this issue:

```cs
namespace Sandbox.WinForms;

public partial class Form1 : Form
{
    public Form1()
    {
        InitializeComponent();

        formsPlot1.Plot.Add.Signal(ScottPlot.Generate.Sin());
        formsPlot1.Plot.Add.Signal(ScottPlot.Generate.Cos());

        formsPlot1.Plot.Axes.Left.TickLabelStyle.FontSize = 64;
        formsPlot1.Plot.Axes.Bottom.TickLabelStyle.FontSize = 64;
    }
}
```

![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/a31ea6bc-0c7a-43ab-88f1-845c3538f82c)

**Comment by swharden** [comment]
Thanks again for reporting this! Fixed in #3487
![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/4a12b3e3-9baa-4723-9fc4-05f6ed93e491)
